lib.php

```diff
@@ -798,6 +812,24 @@ function plagiarism_vericite_ends_with($str, $test) {
     return substr_compare($str, $test, -strlen($test), strlen($test)) === 0;
 }
 
+/**
+ * Time expired when the difference between now and the last time we've checked it, is greater than the cacheTime
+ * @param $timeRetrieved
+ * @param $cacheTime
+ * @return bool
+ */
+function plagiarism_vericite_is_cache_time_expired($timeRetrieved, $cacheTime) {
+    plagiarism_vericite_log("now(): " . time() . " retrieved: " . $timeRetrieved . " cacheTime: " . $cacheTime . " timeLeft: " . plagiarism_vericite_time_left_in_cache($timeRetrieved, $cacheTime));
+    if($timeRetrieved == 0) {
```

Comment:
why is this needed? If timeRetrieved is 0, then time() - 0 should be greater than the cacheTime